### ISSUE TITLE
Legislative changes

### ISSUE THREAD

**Issue by minglis** [ministryofjustice/technical-risk-measures#9]
Is this something we want to capture? If the other things are OK, it shouldn't matter about this as we can address the change, and if the other things are bad, then it's bad and this doesn't have an effect?

**Comment by jennyd** [comment]
GDPR and accessibility are the general ones that have come up so far for this column, so maybe we should just be tracking those individually. If legislation related to the specific service changes, requiring a change to the service, we'd probably consider that a product question rather than a technical risk. Think I've just convinced myself that we should have the separate columns! The date is still relevant for accessibility though (existing vs new sites).
